.changeset/cyan-ears-wonder.md

```diff
@@ -0,0 +1,50 @@
+---
+'merchant-center-application-template-starter': minor
+'@commercetools-frontend/mc-scripts': minor
+'playground': minor
+'@commercetools-website/custom-applications': minor
+---
+
+New CLI options! 🎉
+
+## Loading dotenv files
+
+The `mc-script` CLI now supports loading environment variables from [dotenv files](https://www.npmjs.com/package/dotenv).
+
+Previously, we recommended to use `dotenv-cli` to load environment variables before executing the `mc-scripts` command. For example:
+
+```
+dotenv -c development -- mc-scripts start
+
+NODE_ENV=production dotenv -- mc-scripts compile-html
+
+NODE_ENV=production dotenv -e .env.gcp-production-eu -- mc-scripts compile-html
+```
+
+Now the `mc-scripts` CLI has the dotenv features built-in, so you don't need to install and use `dotenv-cli` anymore.
```

Comment:
Huge congrats on 10 years Nicola! 🏆 

I still remember watching CL and having burgers at Phil‘s place ages ago! ☺️

Reply:
It seems like a looong time ago @dferber90